beta/nncf/tensorflow/sparsity/utils.py

```diff
@@ -42,3 +51,40 @@ def prepare_for_tensorboard(raw_sparsity_statistics):
             sparsity_statistics[base_prefix + key] = value
 
     return sparsity_statistics
+
+
+def strip_model_from_masks(model: tf.keras.Model) -> tf.keras.Model:
+    if not isinstance(model, tf.keras.Model):
+        raise ValueError(
+            'Expected model to be a `tf.keras.Model` instance but got: ', model)
```

Comment:
``ValueError('Expected model to be a `tf.keras.Model` instance but got: ', model)`` ->
``ValueError('Expected model to be a `tf.keras.Model` instance but got {}'.format(type(model)))`` or
``ValueError(f'Expected model to be a `tf.keras.Model` instance but got {type(model)}')``

Reply:
Fixed